xml/System.Diagnostics/ActivityContext.xml

```diff
@@ -216,7 +242,7 @@
         <ReturnType>System.Diagnostics.ActivitySpanId</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The span identifier.</summary>
```

Comment:
><summary>The span identifier.</summary> [](start = 8, length = 39)

The ID of the request as known by the caller.

Reply:
```suggestion
        <summary>The ID of the request as known by the caller.</summary>
```